express-notes-api/server.js

```diff
@@ -8,11 +8,10 @@ app.use(express.json());
 const read = async () => {
   try {
     const data = await readFile('data.json', 'utf8');
-    const parsedData = JSON.parse(data);
-    return parsedData;
+    return JSON.parse(data);
   } catch (err) {
     console.error(err);
-    process.exit(1);
+    status(500).json({ error: 'An unexpected error occurred.' });
```

Comment:
Sorry, you can't actually do that. You need a `res` object to send the response on. Didn't this give a compile error? It looks like there may be a Node `status()` function that has nothing to do with Express or returning responses to the caller.

Two ways to deal with this:
1. Pass `res` into this function so that it can formulate and send a 500 response to the caller
2. Get rid of the try/catch in this function entirely and let the caller's try/catch handle the error